usercp.php

```diff
@@ -2168,6 +2168,83 @@
 	}
 }
 
+// Show remove subscription form when GET method and without valid my_post_key
+if($mybb->input['action'] == "removesubscription")
```

Comment:
Can the GET/POST sections be merged to remove duplicated code?

Reply:
I mainly did this to avoid a huge diff when indenting the section above but I'm more than happy to combine both sections under one if statement.

Reply:
IMO that should be don separately in a future PR for 1.8.24, in the interest of keeping this easy to review.